microservices/google-api/google.googlesearch.sql

```diff
@@ -1,15 +1,15 @@
 DROP TABLE IF EXISTS google.googlesearch;
```

Comment:
@mark-walle : I suggest not including this in the DDL. If someone accidentally runs it, they will delete all of our data. 

Rather, if they want to drop the table they can do that manually. What do you think?

Reply:
@mark-walle : Sounds good.

Reply:
Looks like this was done in a commit below. Resolving the thread.